core/socketio/init.sockets.js

```diff
@@ -0,0 +1,86 @@
+
+// function socketInit() {}
+
+function socketInit() {
+  const store = new Store('../../data/');
+  const { slideId } = window.getUrlVars();
+  store.getSlideCollabDetails(slideId).then(response => {
+    console.log(response[0].collabStatus);
+    let collaborationStatus = response.length > 0 ? response[0].collabStatus : false;
+    if (collaborationStatus) {
+      const socket = io("http://localhost:5000", {
+        // WARNING: in that case, there is no fallback to long-polling
+        transports: ["websocket"], // or [ "websocket", "polling" ] (the order matters)
+        auth: {
+          token: document.cookie.split(';')[1].split('=')[1],
+          slideId: window.getUrlVars().slideId,
+        },
+      });
+  
+      socket.on('connect', function () {
+        // Connected, let's sign-up for to receive messages for this room
+        socket.emit('room', window.getUrlVars().slideId);
+      });
+  
+      socket.on("connection_success", (data) => {
+        console.log('Socket connected!', data);
+      })
+  
+      socket.on("user joined", (data) => {
+        console.log('user joined room: ', data);
+      });
+
+      // Error Handling
+      socket.on("connect_error", err => {
+        console.log('Socket Error: ', err);
+      })
+  
+      const constantMock = window.fetch;
+      window.fetch = function () {
+        // Get the parameter in arguments
+        // Intercept the parameter here
+        if (arguments[2] === true) {
+          console.log('arguments: ', arguments);
+          console.log('argument 2: ', arguments[2]);
```

Comment:
I know this isn't final or anything but just a reminder to remove all unnecessary logs and alerts

Reply:
done